src/services/rawg.api.ts

```diff
@@ -1,11 +1,13 @@
 import axios, { AxiosResponse } from 'axios';
 
+import { GAMEAPI_BASE_URL } from 'react-native-dotenv';
+
 interface ApiHandler {
   get<T>(url: string): Promise<AxiosResponse<T>>;
 }
```

Comment:
nice job. 

nothing to resolve just appreciating good typing ;)

Reply:
I learned from my senpai Breno-dono